### ISSUE TITLE
How to know where the element was dropped?

### ISSUE THREAD

**Issue by hkmsadek** [cameronhimself/vue-drag-drop#43]
I am trying to use in a dynamic area. There I have many divs which are drop-able and I am dragging elements with data. Now I can get the transferred data but I am not being able to know where the data was dropped. Is there any ways to do this? Thanks

**Comment by hkmsadek** [comment]
Thank you for quick replay I was able to solve it using `dragenter` property :)

**Comment by cameronhimself** [comment]
If you need to know which element a draggable was dropped onto, you should be responding to the `drop` event. Anyhow, it sounds like you solved the problem for your use case.

**Comment by liquidvisual** [comment]
@hkmsadek I'm trying to figure out how to do this too, would you mind sharing your method?

I'm trying to figure out how to send a reference of the drop object with the drop event. Is that possible?

**Comment by abhish3kk** [comment]
@cameronhimself Sorry to hijack this. Couldn't find another place to comment. 
`<drop v-for="item in dropzones"
         @drop="handleDrop($event, item)"> // $event is the drag obj, item is the drop obj
      Dropzone
  </drop>`

How can I capture DragEvent in this case? If I use `@drop="handleDrop"`, I can get DragEvent as second parameter like
`hanldeDrop(data, event) {
  console.log(event) //event is dragEvent
}`
Edit: I have nested drop zone and want to use `event.stopPropagation()`.
Edit: Solved it by 
`<drop v-for="item in dropzones"
         @drop="(data, event) => {handleDrop($event, item)}"> // $event is the drag obj, item is the drop obj
      Dropzone
  </drop>`
